scheduler/roles/k8s_start_services/tasks/deploy_k8s_services.yml

```diff
@@ -85,6 +85,16 @@
     -p='[{"op": "replace", "path": "/spec/template/spec/containers/0/imagePullPolicy", "value":"IfNotPresent"}]'
   changed_when: true
 
+# HABANA PLUGIN
+- name: Deploy Habana Device plugin
+  ansible.builtin.command: "kubectl create -f '{{ habana_device_plugin_yaml_url }}'"
+  changed_when: true
+  when:
+    - "'habanalabs-device-plugin-daemonset' not in k8s_pods.stdout"
+    - k8s_version >= minimal_gaudi_k8s_version
+    - accelerator_type is defined
+    - accelerator_type == "habana"
```

Comment:
accelerator_type variable is available to the kude_node group only as it was set  in utils/check_intel_gaudi_device.yml.    In k8s_start_services/tasks/deploy_k8s_services.yaml accelerator_type variable will be always in VARIABLE NOT DEFINED state as this role will be called on kube_control_plane group. Please fix this scenario